tests/test_converters.py

```diff
@@ -91,3 +92,76 @@ def test_valid_python_identifier_for_valid(value: str):
 def test_valid_python_identifier_for_invalid(value: str):
     with pytest.raises(BadArgument, match=f'`{value}` is not a valid Python identifier'):
         asyncio.run(ValidPythonIdentifier.convert(None, value))
+
+
+FIXED_UTC_NOW = datetime.datetime.fromisoformat('2019-01-01T00:00:00')
+
+
+@pytest.mark.parametrize(
+    ('duration', 'expected'),
+    (
+        # Simple duration strings
+        ('1Y', datetime.datetime.fromisoformat('2020-01-01T00:00:00')),
```

Comment:
Originally I was going to suggest changing the `datetime` import for the test module because this quite verbose. But, of course, `datetime` is a sin against PEP8 so it would possibly add confusion if it was imported another way. So I propose you use a [parametrised fixture](https://docs.pytest.org/en/latest/fixture.html#fixture-parametrize) for this, and the fixture takes care of calling `fromisoformat`. I _think_ a parameterised fixture would be the right way to do that.

Reply:
Wouldn't that mean that the parametrized fixture would have to yield both the duration string and the datetime object? When you use parametrized fixtures, your test will be run multiple times with all its parameters, once against each of the values yielded by the parametrized fixture.

Is this only because you don't like the verbosity?

Reply:
Yeah, it would mean returning both. It's meant to replace the current parameter decorator.

> Is this only because you don't like the verbosity?
Pretty much. Just an idea and I don't feel strongly about it. You don't have to do it if you think it's for the worse.

Reply:
I've changed it to use parametrized fixtures.

Reply:
Looks good. You took it a step further and made it easier to add more tests (cause the expected string doesn't need to be calculated by you anymore).